frontend/packages/kubevirt-plugin/src/utils/url.ts

```diff
@@ -45,3 +46,24 @@ export const resolveURL = ({ urlObj, maxHostnameParts, maxPathnameParts }) =>
   urlObj.origin === 'null'
     ? urlObj.href
     : `${resolveOrigin(urlObj, maxHostnameParts)}${resolvePathname(urlObj, maxPathnameParts)}`;
+
+export const getVMWizardCreateLink = (
+  namespace: string,
+  itemName: VMWizardName = VMWizardName.WIZARD,
+  mode: VMWizardMode = VMWizardMode.TEMPLATE,
+  template?: string,
+) => {
+  // Overide mode if name is import.
+  const wizardMode = itemName === VMWizardName.IMPORT ? VMWizardMode.IMPORT : mode;
+  const type = itemName === VMWizardName.YAML ? '~new' : '~new-wizard';
+
+  const params = new URLSearchParams();
+  if (wizardMode !== VMWizardMode.VM) {
+    params.append('mode', wizardMode);
+  }
+  if (template) {
+    params.append('template', template);
+  }
+
+  return `/k8s/ns/${namespace || 'default'}/virtualization/${type}${params ? `?${params}` : ''}`; // covers 'yaml', new-wizard and default
```

Comment:
the params is an object here so it is always true - we should first make it a string and then do the comparison for appending "?...."